coffee_github/src/repository.rs

```diff
@@ -97,14 +97,20 @@ impl Github {
                             debug!("found plugin configuration: {}", conf_str);
 
                             let conf_file = serde_yaml::from_str::<Conf>(&conf_str).unwrap();
+                            let conf_lang = (&conf_file.plugin.lang).to_owned();
+                            plugin_lang = match conf_lang.as_str() {
+                                "py" => PluginLang::Python,
+                                "go" => PluginLang::Go,
+                                "rs" => PluginLang::Rust,
+                                "dart" => PluginLang::Dart,
+                                "js" => PluginLang::JavaScript,
+                                "ts" => PluginLang::TypeScript,
+                                _ => continue,
+                            };
```

Comment:
nice, this is was I was thinking, but in terms of performance this isn't good.

If you look at all the methods, before finding the configuration file we try to guess what kind of language is by iterating across all the plugin files, this is an O(file_num) operation that we can skip by looking before at the conf file and then if the conf file is missing, try to iterate across all the plugin file and guess what the language is.

What do you think?